user_management/ui/views.py

```diff
@@ -27,7 +27,13 @@ class VerifyUserEmailView(VerifyAccountViewMixin, generic.RedirectView):
     permission_denied_class = AlreadyVerifiedException
 
     def get_redirect_url(self, *args, **kwargs):
-        return resolve_url(settings.LOGIN_URL)
+        query_string = getattr(settings, 'VERIFIED_QUERYSTRING', '')
+        url_extra = ''
+
+        if query_string and not self.already_verified:
+            url_extra = '?' + query_string
+
+        return resolve_url(settings.LOGIN_URL + url_extra)
```

Comment:
I think this is leading to - https://opbeat.com/incuna/myday-live/errors/32/

SHould it be return resolve_url(settings.LOGIN_URL) + url_extra